src/Persistence/Array_.php

```diff
@@ -25,11 +24,13 @@ public function __construct(array $data = [])
     }
 
     /**
-     * @deprecated TODO temporary for these:
-     *             - https://github.com/atk4/data/blob/90ab68ac063b8fc2c72dcd66115f1bd3f70a3a92/src/Reference/ContainsOne.php#L119
-     *             - https://github.com/atk4/data/blob/90ab68ac063b8fc2c72dcd66115f1bd3f70a3a92/src/Reference/ContainsMany.php#L66
-     *             remove once fixed/no longer needed
+     * Array of last inserted ids per table.
+     * Last inserted ID for any table is stored under '$' key.
+     *
+     * @var array
      */
+    protected $lastInsertIds = [];
+
     public function getRawDataByTable(string $table): array
```

Comment:
I think you removed the comment in some wrong merge in past, right? If so, introduce it back 1:1.

Reply:
Fixed